kafka-client-lib/src/main/java/io/jenkins/plugins/remotingkafka/builder/SecurityPropertiesBuilder.java

```diff
@@ -0,0 +1,116 @@
+package io.jenkins.plugins.remotingkafka.builder;
+
+import io.jenkins.plugins.remotingkafka.KafkaConfigs;
+import io.jenkins.plugins.remotingkafka.enums.SecurityProtocol;
+
+import java.util.Properties;
+import java.util.logging.Logger;
+
+/**
+ * Builder class to build security properties to be used in Kafka consumer and producer client.
+ */
+public class SecurityPropertiesBuilder {
```

Comment:
In this class it is not clear which properties are required to get `build()` operating correctly. Here the defaults are `null`, and IIUC it may lead to null env vars if the properties are not set. Does not look like an expected behavior.

Maybe it makes sense to support defaults